.github/workflows/prover-testing.yml

```
@@ -1,13 +1,5 @@
 name: Prover testing CI
 
```

```suggestion

on: workflow_call
```

Got _No event triggers defined in `on`_ error otherwise :( See https://github.com/Consensys/linea-monorepo/actions/runs/12782331876